### PR TITLE
fix: APU Bleed Pressure when APU off

### LINKED ISSUE #8009: APU bleed pressure showing 0 PSI when APU is off
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2023-04-27T21:19:23+00:00",
    "ref": "refs/heads/experimental",
    "sha": "c907a0913975558481d0988df291df0d96a030d4",
    "actor": "BlueberryKing",
    "event_name": "manual",
    "pretty_release_name": "experimental:c907a091",
    "version": "v0.10.0-exp.c907a09"
}
```

### Describe the bug

APU bleed pressure shows 0 PSI when the APU is off.
APU BLEED

### Expected behavior

APU bleed pressure to be crossed out

### Steps to reproduce

1) Align IRS
2) Observe APU bleed pressure with the APU off

### References (optional)

https://youtu.be/2SHR2ouBaGc
https://youtu.be/VUjfYWNijA8
image

### Additional info (optional)

_No response_

### Discord Username (optional)

Bruno_pt99#5802

### PR DESCRIPTION
**Pull request by KoruptTinker**
Fixes #8009 

## Summary of Changes
Added a check to see if APU is in normal operation while rendering bleed pressure

## Screenshots

Before 
Screenshot_4

After 
Screenshot_5

## References
https://youtu.be/2SHR2ouBaGc
https://youtu.be/VUjfYWNijA8

Discord username (if different from GitHub):
KoruptedToon#7018

## Testing instructions
1. Load the A32Nx in Cold and Dark State
2. Turn on batteries and connect external power
3. Turn on ANY ADIRS. 
3. Go to APU Page and check bleed pressure. Should reflect XX when APU is turned off. 
4. Should function normally and display numerical values in case APU is turned on.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Luz5020** [comment]
PSI Pressure is only displayed when at least one ADIRS unit is functional, so it can show XX under APU operation, might want to mention that in the test instructions

**Review by 2hwk — APPROVED** [review]

File: fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx [hunk 1]
```diff
@@ -168,9 +169,9 @@ const ApuBleed = ({ x, y } : ComponentPositionProps) => {
                 <text
                     x={44}
                     y={48}
-                    className={`FontLarger Right ${adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
+                    className={`FontLarger Right ${apuN.isNormalOperation() && adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx:172]: Up above replace adir1ModeSelector knob with:
```ts
const correctedAverageStaticPressure = useArinc429Var('A32NX_ADIRS_ADR_1_CORRECTED_AVERAGE_STATIC_PRESSURE');
```
Then above `return (`
```ts
// FIXME should be APU bleed absolute pressure label from SDAC
const apuBleedPressAvailable = apuN.isNormalOperation() && correctedAverageStaticPressure.isNormalOperation();
```
Then:
```suggestion
                    className={`FontLarger Right ${apuBleedPressAvailable ? 'Green' : 'Amber'}`}
```

File: fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx [hunk 2]
```diff
@@ -168,9 +169,9 @@ const ApuBleed = ({ x, y } : ComponentPositionProps) => {
...
                 >
-                    {adir1ModeSelectorKnob === 1 ? displayedBleedPressure : 'XX'}
+                    {apuN.isNormalOperation() && adir1ModeSelectorKnob === 1 ? displayedBleedPressure : 'XX'}
```
Comment (tracernz) [inline fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx:174]: With above changes:
```suggestion
                    {apuBleedPressAvailable ? displayedBleedPressure : 'XX'}
```

**Review by tracernz — COMMENTED** [review]
To fix #8001 as well.

**Comment by patmack14** [comment]
> To fix #8001 as well.

Is #8001 closed then too?

**Comment by tracernz** [comment]
> Is #8001 closed then too?

Not at this stage as the suggested changes have not been applied.

**Pull request closed without merging**